elasticsearch/manifests/elasticsearch.yaml

```diff
@@ -390,9 +383,18 @@ spec:
       
       affinity:
         podAffinity:
-          
+
         podAntiAffinity:
-          
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
From what I understand this _could_ still result in pods being scheduled onto the same node (link: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#nodeaffinity-v1-core, emphasis added by me)

> The scheduler will **prefer** to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions

is this ok?